_editorial_policies/01_bylaws.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: single
+sidebar:
+  nav: policies_bylaws.md
+title: Bylaws of the Living Journal of Computational Molecular Science
+excerpt: Bylaws of LiveCoMS.
+permalink: /policies/livecoms_bylaws/
+---
+
+1. Organization Title and Publication
+
+   The Journal shall be called the Living Journal of Computational
+   Molecular Science, hereafter referred to as LiveCoMS.
+
+   LiveCoMS shall produce one publication, the Living Journal of
+   Computational Molecular Science, with articles released on a rolling
+   basis online. LiveCoMS provides a peer-reviewed home for manuscripts
+   which share best practices in molecular modeling and simulation. These
+   works are living documents, regularly updated with community input,
+   and can include living or perpetual reviews, tutorials, comparisons
+   between software packages, and other documents which aim to improve
+   the studies in the field and require ongoing updates.
+
+1. Editorial Board
+
+   The editorial board shall consist of three Managing Editors, any
+   number of Associate Editors, and a Section Lead Editor for each type
+   of document published in LiveCoMS, and any number of Associate
+   Editors. Section Lead Editors may be either Managing Editors or
+   Associate Editors.
+
+   1. Responsibilities
+       1. The Managing Editors coordinate the operation of the Journal and
+          the finances of the journal, maintain the online presence.  Tasks
+          may be divided up among managing editors as they see fit, and specific
+          individual tasks may be assigned to individual associate editors.
+       1. Section Lead Editors handles submitted manuscripts and assigns
+          all manuscripts to associate lead editors. They make decisions on
+          presubmission letters on about proposed manuscripts, in which they
+          may be advised by the associate editors. They make final decisions
+          based on recommendations from the associate editors, and communicate
+          them to the authors.
+       1. Associate Editors assign manuscripts to reviewers, collect
+          these reviews, and make recommendations to the Section Lead
+          Editors about acceptance, rejection, or requests for revision.
+
+   1. Selection process
+       1. Members of the editorial board will be elected as needed by a two-thirds vote of the editorial board.
```

Comment:
I think it best if the board has flexibility.  In terms of as needed, if the editorial board things they need more!  But we can rephrase.  "as the editorial board deems useful for the Journal"?  I kind of like something in there like that to indicate they can elect more if they want.

Reply:
I like flexibility too, which is part of why I didn't like the word "needed" as (to me) it suggested there would be some factor dictating how many we need, as opposed to just what I think we will want to do, which is say, "Hey, X is interested in helping with this, so can we fit her on the board?" or "Seems like we're getting deluged with requests so it might be good to have a couple more people on the board," etc.